doc/release-notes.md

```diff
@@ -1,5 +1,4 @@
-(note: this is a temporary file, to be added-to by anybody, and moved to
-release-notes at release time)
+*(Note: this is a temporary file to be added-to by anybody. See [/doc/developer-notes.md#release-notes](/doc/developer-notes.md#release-notes) for the process. After branching off for a major version release of Bitcoin Core, release notes are drafted in a collaborative wiki at https://github.com/bitcoin-core/bitcoin-devwiki/wiki/*version*-Release-Notes-Draft). Before the final release, the notes need to be moved to [/doc/release-notes/](/doc/release-notes/).*
```

Comment:
Could make this multiline?

```suggestion
*This is a temporary file to be added-to by anybody. See
[/doc/developer-notes.md#release-notes](/doc/developer-notes.md#release-notes)
for the process.*

*After branching off for a major version release of Bitcoin Core, release notes
are drafted in a collaborative wiki at*
https://github.com/bitcoin-core/bitcoin-devwiki/wiki/*version*-Release-Notes-Draft.
*Before the final release, the notes need to be moved back to this git
repository.*
```

Reply:
Done, and used your changes to the writing. Also made the wiki link point to a functional url and did another writing pass.